src/components/DiffOutput.tsx

```diff
@@ -22,42 +22,22 @@ export type DiffOutputProps = DiffArrowProps & OutputProps & ElementProps & Styl
 export const DiffOutput = ({
   className,
   direction,
-  fractionDigits,
   hasInvalidNewValue,
   sign,
-  tag,
-  type,
-  useGrouping,
   withDiff,
   layout = 'row',
-  showSign,
-  withBaseFont,
 
   value,
   newValue,
+
+  ...outputProps
```

Comment:
qq: what happens if DiffOutput has an added prop that overrides an output prop? It wouldn't get passed down in that case right?

Reply:
yeah we'd have to manually add it after the {...outputProps} I think